src/util.ts

```diff
@@ -4,11 +4,10 @@ import { existsSync } from 'fs-extra';
 import path = require('path');
 import { window, workspace } from 'vscode';
 import winreg = require('winreg');
-export let config = workspace.getConfiguration('r');
```

Comment:
how is just change to
```ts
export const config = (function() {
    return workspace.getConfiguration('r');   
});
```
It will reduce whole of changes.

Reply:
@Ikuyadeu Exactly, that's what I was thinking of.

Reply:
@Ikuyadeu 
```typescript
export const config = (function() {
    return workspace.getConfiguration('r');   
});
```
is better than
```typescript
export function config() {
    return workspace.getConfiguration('r');
}
```
?

Reply:
Yes this looks better.

Reply:
I agree with you.

Reply:
Okay, I will change to this.

```typescript
export function config() {
    return workspace.getConfiguration('r');
}
```

The commits are becoming a bit messy. I would like to tidy up using REBASE and FORCE PUSH. @Ikuyadeu @renkun-ken Do you approve rebase and force push here?

Reply:
In this case, `Squash and Merge` is better, it will combine your commits to the one.
After your change, I will merge it.

Reply:
@Ikuyadeu Okay, no rebase. Thanks in advance for squash and merge.

I will add the commit for `config()`.

Reply:
I'm totally fine with both.